### DISCUSSION TITLE
Feedback: Unreliable synchronization of longer track recordings

### DISCUSSION THREAD

**Discussion by CM000n** [DurgNomis-drol/ha_toyota#204] [Show and tell]
Hi, this is not so much a bug but an observation I made during our Italy vacation. During the vacation, I noticed that some routes and information such as current fuel level are simply no longer synchronized with Toyota Connected Services.
After some searching, I found out that I am not alone in this. There is a discussion about this in a German Corolla forum, where users describe that on longer routes (about 300km) the synchronization becomes unreliable.

Here ist the Link to the corresponding forum Thread:
https://www.toyota-corolla-forum.de/forum/thread/338-myt-app-synchronisation/?pageNo=1

In this discussion the assumption was made that the vehicle data is probably cached in the on-board computer while driving and only transferred after the vehicle has come to a standstill. If the vehicle data has not been transferred by the next trip, it will probably be discarded.
Interestingly, after a user contacted Toyota Germany, it emerged that 3G SIM cards from Vodafone are installed in our vehicles. Since the 3G network has already been abolished in many countries, the vehicles have to fall back on the 2G network in these cases. This makes it clear that larger data volumes can only be transmitted very poorly or not at all.

Since this synchronization behavior could possibly also result in bug reports for this custom component, I thought I would write it here as information. So please check if the data is available in your MyT app before you open a bug here. If not, you have to open one at Toyota itself ;-)

**Comment by DurgNomis-drol** [comment]
Thanks for taking your time to describe this here.

In the last month or so, I did notice that some of my trips where not logged, and those where all short trips (Under 50 km). So I don't think this is exclusive to long trips, but it might happen easier with them. 

Please do update me if you find out more information on your German forum 😊

**Comment by homonto** [comment]
unfortunately sooner than later both 2G and 3G will be obsolete - all goes to 4G now - I am wondering if Toyota will replace SIM cards - it might require also changing the modem
